subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -184,11 +184,12 @@ static void handle_wifi_raw_scan_result(struct net_mgmt_event_callback *cb)
 	int rssi;
```

Comment:
How is this reducing code footprint, you have just an extra variable in the code which is probably optimized out by the compiler although I did not check this.

Reply:
Hi @jukkar, FWIW, I built **"samples/net/wifi"** on the **"disco_l475_iot1"** board, using various optimizations detailed in the table below.

| OPTIMIZATIONS  | FLASH SIZE (BEFORE) | FLASH SIZE (AFTER) | REDUCE (AFTER - BEFORE) |
| ------------- | --------------------------- | ---------------------------------- | ---- |
| SIZE_OPTIMIZATIONS  | 175,248 B | 175,176 B | -72 B |
| SPEED_OPTIMIZATIONS  | 202,552 B | 202,472 B | -80 B |
| DEBUG_OPTIMIZATIONS  | 209,396 B | 209,372 B | -24 B |

---
When `SIZE_OPTIMIZATIONS=y`, the code footprint of `wifi_shell.c` is reduced by **72 bytes**, 
from 9,084 bytes to 9,012 bytes.